src/css/options.css

```diff
@@ -1,29 +1,122 @@
 body {
+  --grey10: #e7e7e7;
+
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
`system-ui` is the equivalent of `-apple-system` on Linux, and results in the same font that's used by the current desktop theme by default.